<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HeaderPopup</class>
 <widget class="QFrame" name="HeaderPopup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>122</width>
    <height>149</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Header Popup</string>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QGridLayout" name="popupLayout">
   <property name="leftMargin">
    <number>8</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <property name="horizontalSpacing">
    <number>1</number>
   </property>
   <property name="verticalSpacing">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <widget class="QCheckBox" name="fileNameCheckBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>File Name</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="extCheckBox">
     <property name="text">
      <string>Extension</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="sizeCheckBox">
     <property name="text">
      <string>Size</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="dateCheckBox">
     <property name="text">
      <string>Date</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="ownerCheckBox">
     <property name="text">
      <string>Owner</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="permsCheckBox">
     <property name="text">
      <string>Permissions</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QFrame" name="separator">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QPushButton" name="closeButton">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>18</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>18</height>
      </size>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
